ui/StatusQ/tests/tst_Aggregator.cpp

```diff
@@ -0,0 +1,148 @@
+#include <QtTest>
+#include <QAbstractItemModel>
```

Comment:
```suggestion
#include <QAbstractListModel>
```
similarly in other tests